demo/apiExamples.md

```diff
@@ -7,20 +7,23 @@ The auro-toast element provides users a way to ... (it would be great if you fil
 
 ## Attributes
 
-| Attribute | Type      | Description                                      |
-|-----------|-----------|--------------------------------------------------|
-| [fixed](#fixed)   | `Boolean` | Uses fixed pixel values for element shape        |
-| [type](#type)    | `String`  | Component will render visually based on which type value is set; currently supports `error`, `success` |
+| Attribute | Type      | Description                               |
+|-----------|-----------|-------------------------------------------|
+| [fixed](#fixed)   | `Boolean` | Uses fixed pixel values for element shape |
```

Comment:
This attr is no longer supported. Please remove the following line from ./src/auro-toast.js

```
@attr {Boolean} fixed - Uses fixed pixel values for element shape
```